brian2/stateupdaters/base.py

```diff
@@ -127,10 +138,16 @@ def apply_stateupdater(equations, variables, method):
             if the_method is None:
                 raise ValueError(('No stateupdater that is suitable for the '
                                   'given equations has been found.'))
-            logger.info('Using state updater: %r' % the_method)
+            if group_name is not None:
+                msg_text = ("No numerical integration method specified for group "
+                            "'{group_name}', choosing method '{method}'.")
+            else:
+                msg_text = ("No numerical integration method specified, "
+                            "choosing method '{method}'.")
+            logger.info(msg_text.format(group_name=group_name,
+                                        method=the_method), 'method_choice')
```

Comment:
As long as the info log doesn't get too overburdened that's OK (otherwise just the total time is fine I think). Certainly it should only be one line at most.